libzeth/core/field_element_utils.tcc

```diff
@@ -34,16 +35,59 @@ libff::bigint<FieldT::num_limbs> bigint_from_hex(const std::string &hex)
 }
 
 template<typename FieldT>
-std::string field_element_to_hex(const FieldT &field_el)
+std::string base_field_element_to_hex(const FieldT &field_el)
```

Comment:
Yes we can do that, I initially implemented a single recursive function which was calling itself on all extension field coefficients down until the base case/stopping condition which was when `FieldT::extension_degree == 1` (i.e. no more tower levels to navigate -> base field).

However, since our twist fields (in the current state of our projects) are small extensions (i.e. Fp, Fp^2 or Fp^3 depending on the curves we use), these extensions didn't require any towering, and I thus decided to get away from this recursive function to aim for something simpler with these 2 functions. Happy to go back to the single - recursive function case though